pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -19,10 +19,12 @@
 , useFabricmanager ? false
 , ibtSupport ? false
 
-, prePatch ? ""
+, prePatch ? null
 , postPatch ? null
 , patchFlags ? null
 , patches ? [ ]
+, preInstall ? null
```

Comment:
You will need to add `inherit preInstall postInstall;` to the derivation for this to work.

Reply:
I did test the package, but I probably forgot to also remove my temporary patch in my configuration.nix when testing. Will test again, because what you're saying makes sense

Reply:
Ok, looks like I didn't actually tested it even though I thought I did ... Anyway indeed this was missing, now that I fixed it I indeed no longer see any `lib/tls` directory in the nvidia-x11 package, as it should. However I couldn't test if it properly works because for some reason with that testing profile X puts the nvidia-x11 modules after the regular xorg modules, which causes the xorg GLX module to be loaded instead of the nvidia one and thus makes nvidia fails. But it should work ?